x/application/types/genesis_test.go

```diff
@@ -3,6 +3,7 @@ package types_test
 import (
```

Comment:
We don't. Will move `x/application/types/genesis_test.go` to `x/application/module/genesis_test.go` since this where the new CosmosSDK wants us to have them.